pyingest/parsers/default.py

```diff
@@ -57,3 +62,18 @@ def _attr(self, e, k, d=''):
             return d
         else:
             return d
+
+class BaseBeautifulSoupParser(BaseXmlToDictParser):
```

Comment:
Agreed, however I'd like to have a Base bs parser object for parity with xmltodict rather than calling bs4.BeautifulSoup() within jats.py.  TBH, I would like to rewrite all the parsers to use BeautifulSoup eventually.